clang/test/SemaSYCL/reqd-sub-group-size-amd_64.cpp

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple amdgcn-amd-amdhsa -target-cpu gfx90a -internal-isystem %S/Inputs -std=c++2b -verify %s
+
+// Sub-group size is optimized for 32, warn (and ignore the attribute) if the
```

Comment:
This comment is incorrect for this test right?

Reply:
Yes, copy past error, fixed now.